### DISCUSSION TITLE
Ktlint command to fix the issue

### DISCUSSION THREAD

**Discussion by rishujam** [oppia/oppia-android#4773] [Q&A (Other)] [answered]
![Screenshot 2022-11-27 235433](https://user-images.githubusercontent.com/74773876/204442810-d48b87d2-d891-42c3-9f69-1559c6a56fa1.png)
So do I need to open the oppia-android-tools folder and open cmd there? And then write the file location which I need to fix?
@BenHenning

**Comment by BenHenning** [comment, marked as answer]
You need to run the command, as written in the output, in the root of the Oppia Android repository.
